Libraries/Components/ViewPager/ViewPagerAndroid.android.js

```diff
@@ -151,6 +152,13 @@ var ViewPagerAndroid = React.createClass({
   setPage: function(selectedPage) {
     this.setState({
       selectedPage,
+      animatePageTransition: true,
```

Comment:
This has a hidden side effect of setting `animatePageTransition` for all future updates of `selectedPage`.

How about making this explicit and exposing a `animateSetPage` prop? (Feel free to come up with a better name :))